pipeline/common_stages.groovy

```diff
@@ -453,7 +457,7 @@ def deploy_mig_controller_on_both(
         ansiColor('xterm') {
           ansiblePlaybook(
             playbook: 'mig_controller_deploy.yml',
-            extras: "-e mig_controller_host_cluster=${mig_controller_dst} -e mig_controller_ui=${MIG_CONTROLLER_UI}",
+            extras: "-e mig_controller_image=${mig_controller_img} -e mig_controller_version=${mig_controller_tag} -e mig_controller_host_cluster=${mig_controller_dst} -e mig_controller_ui=${MIG_CONTROLLER_UI}",
```

Comment:
allow overriding mig-controller image in MigrationController configuration - custom PRs